driver/private_include/camera_common.h

```diff
@@ -3,7 +3,7 @@
 #include <stdint.h>
 #include <stddef.h>
 #include <stdbool.h>
-#include "rom/lldesc.h"
+#include "esp32/rom/lldesc.h" 
```

Comment:
why? it should work without `esp32/`. When I add support for the new chips, this header will not resolve.

Reply:
ok, I did some research and here is how you should handle all headers that require `esp32/` change:
```cpp
#include "esp_system.h"
#ifdef ESP_IDF_VERSION_MAJOR // IDF 4+
#if CONFIG_IDF_TARGET_ESP32 // ESP32/PICO-D4
#include "esp32/rom/lldesc.h"
#endif
#else
#include "rom/lldesc.h" // ESP32 Before IDF 4.0
#endif
```

Reply:
I actually took care of that, so revert the renames and rebase from master. Will make your PR cleaner too